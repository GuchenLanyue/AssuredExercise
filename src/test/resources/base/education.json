{"status":1,"msg":"success","value":{"1":"\u4e0d\u9650","2":"\u521d\u4e2d","3":"\u9ad8\u4e2d","4":"\u4e2d\u6280","5":"\u4e2d\u4e13","6":"\u5927\u4e13","7":"\u672c\u79d1","8":"\u7855\u58eb","9":"\u535a\u58eb"}}