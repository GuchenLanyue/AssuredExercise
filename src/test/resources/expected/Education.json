{
  "case": "base1",
  "expected": [
    {
      "root": "value",
      "values": {
        "name": "微迅科技大学",
      }
    }
  ]
}